spring-integration-mail/src/main/resources/org/springframework/integration/mail/config/spring-integration-mail-5.0.xsd

```diff
@@ -398,4 +400,23 @@
 		</xsd:attribute>
 	</xsd:attributeGroup>
 
+	<xsd:attributeGroup name="simpleContentGroup">
+		<xsd:attribute name="simple-content">
```

Comment:
M-m-m. Can't we just move this new common attribute to an existing `inboundMailAdapterType` type?